mojap_metadata/converters/glue_converter/__init__.py

```diff
@@ -344,6 +391,44 @@ def generate_from_meta(
         )
         return spec
 
+    def generate_to_meta():
+        raise NotImplementedError()
+
+
+class GlueTable:
+
+    gc = GlueConverter()
+
+    @classmethod
```

Comment:
I think this should produce an object rather than just be a class? 

If it was an object:
- It follows the same protocols as other converters
- You should be able to edit the glue converter object in the glue table class. The glue converter object has loads of settings and atm as an object in a class you're changing the same object at runtime? (I think anyway)